src/explore-education-statistics-frontend/src/modules/data-catalogue/components/DataSetFileApiVersionHistory.tsx

```diff
@@ -99,4 +123,22 @@ export default function DataSetFileApiVersionHistory({
       </LoadingSpinner>
     </DataSetFilePageSection>
   );
+  function showCurrentTag(
```

Comment:
Not a biggie, but I found the name of this a bit confusing, I was expecting it to return a `<Tag>` element.

It seems like it's something that might be useful in other places so maybe it could be moved to a util and just return a boolean? It could then be unit tested as well.